lib/kramdown-rfc2629.rb

```diff
@@ -539,7 +539,8 @@ def svg_tool_process(t, svg_opt, txt_opt, result)
           err << err1.to_s
         when "math"
           result1, err, _s = Open3.capture3("tex2svg --font STIX --speech=false#{svg_opt} #{Shellwords.escape(' ' << result)}");
-          result, err1, _s = Open3.capture3("asciitex -f #{file.path}#{txt_opt}")
+          math = File.read(file.path)
+          result, err1, _s = Open3.capture3("utftex '#{math}'#{txt_opt}")
```

Comment:
Thanks!  We probably want to do a shell escape instead of '#{math}', which will fail in interesting ways.
(I can do that after merging, if you want.)

Reply:
Please do - ruby newbie